pkg/objectstore/s3/s3.go

```diff
@@ -81,7 +81,9 @@ func GetObjLockInfo(backupLocation *stork_api.BackupLocation) (*common.ObjLockIn
 			// When a minio server doesn't have object-lock implemented
 			// then Minio backed backuplocation throws this error for
 			// normal buckets too hence we need to ignore this for now.
-			if awsErr.Code() == "ObjectLockConfigurationNotFoundError" || awsErr.Code() == "MethodNotAllowed" {
+			if awsErr.Code() == "ObjectLockConfigurationNotFoundError" ||
+				awsErr.Code() == "MethodNotAllowed" ||
```

Comment:
Can u please add a comment saying we are getting NoSuchBucket from minio and ObjectLockConfigurationNotFoundError error from aws S3 when the bucket does not exists